test/integration/scheduler/podsready/scheduler_test.go

```diff
@@ -248,9 +259,24 @@ var _ = ginkgo.Describe("SchedulerWithWaitForPodsReady", func() {
 			time.Sleep(podsReadyTimeout)
 			ginkgo.By("finish the eviction")
 			util.FinishEvictionForWorkloads(ctx, k8sClient, prodWl)
-			ginkgo.By("verify the 'prod' workload gets re-admitted once")
+			ginkgo.By("pending by backoff")
+			util.ExpectWorkloadsToBePending(ctx, k8sClient, prodWl)
```

Comment:
This could be flaky. Since there is the first backoff re-queueing, it takes just 1 sec + jitter.
Maybe, we should increase the factor and duration here: https://github.com/kubernetes-sigs/kueue/blob/13bb0d6a7514a97feb4fae8c800a2b4febc0d32f/pkg/controller/core/workload_controller.go#L383-L384

Although it makes the integration tests more duration.

Reply:
I would just remove this check and leave a comment.

Reply:
That makes sense.